CHANGELOG.rst

```diff
@@ -2,6 +2,100 @@
 Release Notes
 =============
 
+.. _Release Notes_7.16.0:
+
+7.16.0 / 6.16.0
+======
+
+.. _Release Notes_7.16.0_Prelude:
+
+Prelude
+-------
+
+This release introduces major version 7 of the Datadog Agent, which starts at v7.16.0. The only change from Agent v6 is that
+v7 defaults to Python 3 and only includes support for Python 3. Before upgrading to v7, confirm that any
+custom checks you have are compatible with Python 3. See this ``guide <https://docs.datadoghq.com/agent/guide/python-3/>``_
+for more information.
+
+Except for the supported Python versions, v7.16.0 and v6.16.0 have the same features.
+
+
```

Comment:
actually, we seem to be missing the prelude that links to the tag on the `integrations-core` repo and mentions the release date?

Whether it was added or not in a separate release note file (as it's supposed to be), feel free to just add the relevant content directly in the changelog here. No need to add the file in `releasenotes/` now if it's not there